ShoppingCart-topic1/src/List.java

```diff
@@ -0,0 +1,8 @@
+public interface List {

```

Comment:
Please remove keypoints related to previous topics adding it on .gitignore or removing the file in the commit using "git rm 'filename.java'" (if the file was removed from the workspace), "git rm 'folderName' -r" (if a folder was removed), "git rm --cached 'filename.java'" (if you are not able to remove the file from the disk),  "git rm --cached 'foldername' -r" (if you are not able to remove the folder from the disk),